bcda/auth/client/oktaclient.go

```diff
@@ -149,6 +164,54 @@ func (oc *OktaClient) AddClientApplication(localID string) (string, string, erro
 	return clientID, clientSecret, nil
 }
 
+func (oc OktaClient) RequestAccessToken(creds Credentials) (OktaToken, error) {
+	requestID := uuid.NewRandom()
+
+	tokenURL := fmt.Sprintf("%s/oauth2/%s/v1/token", oktaBaseUrl, oktaServerID)
+	req, err := http.NewRequest("POST", tokenURL, nil)
+	if err != nil {
+		return OktaToken{}, err
+	}
+
+	req.Header.Add("Accept", "application/json")
```

Comment:
lets reuse the addRequestHeaders() func i made here
https://github.com/CMSgov/bcda-app/blob/7ee08580546daba2c233a0c95048764dc2ec2cf4/bcda/auth/client/oktaclient.go#L288-L292

Reply:
The Content-Type is actually different in this code.  I'm not sure that happens often enough to add complexity to your addRequestHeaders() func.